CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 
 ## Prerelease
 
+- Add Definition List component.
```

Comment:
nit: in the past we've added the [type of change](https://keepachangelog.com/en/1.0.0/#how) in the changelog